docs/installation.md

```diff
@@ -30,6 +30,12 @@ Determine your base installation option.
 * A **namespace install** only executes workflows in the namespace it is installed in (typically `argo`). Look for `namespace-install.yaml` in the [release assets](https://github.com/argoproj/argo-workflows/releases/latest).
 * A **managed namespace install**: only executes workflows in a specific namespace ([learn more](managed-namespace.md)).
 
+!!! Warning "namespace install vs. managed namespace install"
```

Comment:
```suggestion
!!! Info "namespace install vs. managed namespace install"
```

I think [info](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) makes more sense here as opposed to a warning. There is nothing to warn about, this is just descriptive